### PR TITLE
Added the server option `host`.

### PR DESCRIPTION
**Pull request by PascalTurbo**
With this option it's possible to bind yard to an other interface than localhost, 0.0.0.0 for example.

### CODE REVIEW

File: CHANGES.markdown [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 2.1.5 (2016-08-03)
```
Comment (panthomakos) [inline CHANGES.markdown]: You can leave this as `# master (unreleased)`. I'll rev the version for a new gem.

File: lib/guard/yard/version.rb [hunk 2]
```diff
@@ -1,5 +1,5 @@
 module Guard
   module YardVersion
-    VERSION = '2.1.4'.freeze
+    VERSION = '2.1.5'.freeze
```
Comment (panthomakos) [inline lib/guard/yard/version.rb]: Ditto here - don't worry about changing this.

**Comment by panthomakos** [comment]
ping - any updates?

**Comment by PascalTurbo** [comment]
I’m sorry, I didn’t recognize, that I have to modify the pull request.

Once my internet connection is working again, I will do this.

**Comment by PascalTurbo** [comment]
Finaly, I did it.

**Comment by panthomakos** [comment]
Thanks for the contribution!

**Pull request merged**